.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
Is there a reason we use `https://go.microsoft.com/fwlink/?LinkId=827846` instead of the URL it redirects to (which looks a bit cleaner) https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions?

Reply:
It's a standard practice so that if we need to update the doc URL for recommended extensions, we can just update the Link versus every single copy of a devcontainer out there (past and present)

Reply:
A link like this stays evergreen with any URL changes - assuming we update it to a working link :)

Reply:
Got it - thanks for the explanation :)